lib/capybara/server.rb

```diff
@@ -5,6 +5,16 @@
 module Capybara
   class Server
     class Middleware
```

Comment:
Probably - I'm just not convinced I like the whole solution - would much prefer if selenium allowed us to preset a cookie before the first visit